test/core/integration/concrete/lockup-dynamic/get-stream/getStream.t.sol

```diff
@@ -4,14 +4,11 @@ pragma solidity >=0.8.22 <0.9.0;
 import { Errors } from "src/core/libraries/Errors.sol";
 import { LockupDynamic } from "src/core/types/DataTypes.sol";
 
-import { LockupDynamic_Integration_Concrete_Test } from "../LockupDynamic.t.sol";
-
-contract GetStream_LockupDynamic_Integration_Concrete_Test is LockupDynamic_Integration_Concrete_Test {
-    uint256 internal defaultStreamId;
+import { LockupDynamic_Integration_Shared_Test } from "../LockupDynamic.t.sol";
 
+contract GetStream_LockupDynamic_Integration_Concrete_Test is LockupDynamic_Integration_Shared_Test {
     function setUp() public virtual override {
```

Comment:
You can remove this setup entirely from non abstract contracts if it does not contain any necessary logic.